generators/app/templates/app/middlewares/docsAuth.ejs

```diff
@@ -0,0 +1,19 @@
+const { decode } = require('jwt-simple');
+
+const logger = require('../logger');
+
+const isDocumentationAuthenticated = async (req, res, next) => {
+  const token = req.headers['x-access-token'];
```

Comment:
put other name to this header because this one is too generic. something like `documentation-token`